inventory/packages/apt_deb.go

```diff
@@ -61,6 +62,20 @@ func InstallAptPackages(pkgs []string) error {
 	return err
 }
 
+// InstallAptPackagesIndividually installs apt packages individually.
+func InstallAptPackagesIndividually(pkgs []string) error {
```

Comment:
is it possible to not introduce this new method here in packge manager and instead just introduce this fallback in "policies" where we call "InstallAptPackages". 
I am asking this because this is supposed to be just go implementation of package managers(or atleast that is how i see this package)

Reply:
Done